source/modules/collection-api/sale/overview.md.erb

```diff
@@ -1,19 +1,20 @@
 ## Sale Activity
 
-The following functions provide Inspetor with information on sales that happen within your platform. The main activities that we track are the following:
+You can provide sale-related activity via the REST API. The `/sales` endpoint supports `POST` and `PUT` operations with the following arguments:
```

Comment:
The `curl` example commands are hitting `/sale` instead of `/sales`

Reply:
Ah right--it should be `/sales`